spec/unit/exception_handling/sensu_spec.rb

```diff
@@ -6,21 +6,21 @@ module ExceptionHandling
   describe Sensu do
     context "#generate_event" do
       it "create an event" do
-        expect(ExceptionHandling::Sensu).to receive(:send_event).with(name: "world_is_ending", output: "stick head between knees and kiss ass goodbye", status: 1)
+        expect(ExceptionHandling::Sensu).to receive(:send_event).with({ name: "world_is_ending", output: "stick head between knees and kiss ass goodbye", status: 1 })
```

Comment:
These tests were failing in other PRs as of mid December. So not related to my changes, but I fixed them anyways.